code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -714,3 +714,90 @@
 		segment_length = armsy.get_length()
 		qdel(armsy)
 		return
+
+/obj/effect/proc_holder/spell/pointed/void_blink
+	name = "Void Phase"
+	desc = "Let's you blink to your pointed destination, causes 3x3 aoe damage bubble around your pointed destination and your current location"
+	invocation_type = INVOCATION_WHISPER
+	invocation = "RE'L'TY PH'S'E"
+	clothes_req = FALSE
+	range = 9
+	action_background_icon_state = "bg_ecult"
+	charge_max = 300
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "voidblink"
+	selection_type = "range"
+
+/obj/effect/proc_holder/spell/pointed/void_blink/can_target(atom/target, mob/user, silent)
+	. = ..()
+	if(get_dist(get_turf(user),get_turf(target)) < 2 )
+		return FALSE
+
+/obj/effect/proc_holder/spell/pointed/void_blink/cast(list/targets, mob/user)
+	. = ..()
+	var/target = targets[1]
+	var/turf/targeted_turf = get_turf(target)
+
+	playsound(user,'sound/magic/voidblink.ogg',100)
+	playsound(targeted_turf,'sound/magic/voidblink.ogg',100)
+
+	new /obj/effect/temp_visual/voidin(user.drop_location())
+	new /obj/effect/temp_visual/voidout(targeted_turf)
+
+	for(var/mob/living/living_mob in range(1,user)-user)
+		if(IS_HERETIC(living_mob) || IS_HERETIC_MONSTER(living_mob))
+			continue
+		living_mob.adjustBruteLoss(40)
+
+	for(var/mob/living/living_mob in range(1,targeted_turf)-user)
```

Comment:
step 1: unequip anything that makes you look like a heretic
step 2: shovespam an officer until they staminacrit you and try to cuff you
step 3: press the "okay" of the "nothin' personnel... kid..." textbox that you've pre-typed
step 4: teleport 1 square behind them with void blink (which should be possible, as the minimum range of this spell is 2 tiles)
step 5: cast void pull
step 6: laugh, because you've just dealt 110 brute damage (that ignores armor!) to the poor sap who made the grave error of standing still next to a staminacritted, possibly even cuffed target

Reply:
IMO, the best way to fix this is NOT to just increase the minimum range of this spell up to 3 tiles, since that'd just make this spell quite clunky.

Instead, properly implement a check to make sure you're not damaging the same mob twice. Off the top of my head, you create a list, and add each mob that should be damaged by the spell to the list if they aren't already on it. Then, you can just iterate through the list and deal 40 damage to each mob in it. With that solution, you can even remove the unintuitive/awkward minimum range limitation of the spell!

Reply:
im too lazy to implement the list method, setting the minimum range to 3 is satisfactory.

Reply:
oh come ON

I'll write the code myself if I have to- please don't gimp void blink like this.